src/shared/application/ports/out/uploads-backend.ts

```diff
@@ -0,0 +1,3 @@
+export interface UploadsBackend {
```

Comment:
dont worry about it 
![image](https://github.com/user-attachments/assets/681756b2-26dc-4af0-87bc-c09c5178b8ca)